content/best_practices.adoc

```diff
@@ -56,7 +56,48 @@ Some operations require logging/auditing to allow to understand what happened du
 
 == Performance
 
-The reviewer must try to detect if the fix/feature could have significant bad impacts on the performances of the application. Performances issues can be expensive to find and fix, so it is important to raise any concern on this topic at this stage. Here are some examples of performance issues causes: too many database requests, slow database query, missing index in the database, too many HTTP requests, …​
+The reviewer must try to detect if the fix/feature could have significant bad impacts on the performances of the application. Performances issues can be expensive to find and fix, so it is important to raise any concern on this topic at this stage. Here are some examples of performance issues causes: too many database requests, slow database query, missing index in the database, too many HTTP requests, …
+
+=== Front-End Performances
+
+​In Front-End developing, it's important to consider best solutions that reduces Memory & CPU consumption in Browser. In fact, a Web page executes only one single thread for all Vue apps, so it's important to avoid adding a lot of stuff to do when mounting a Vue application (https://www.red-gate.com/simple-talk/development/dotnet-development/javascript-single-threaded/[Read more about JS lifecycle]). In fact, when adding a heavy computing statements in page loading phase, it will *pause* mounting all other Vue applications until the computing is finished, because of single-threaded execution architecture of Web pages. Consequently, having one Vue app that isn't sufficiently performant in a web page, especially for a TopBar application (that is present in all pages), will degrade the rendering performances of the whole page.
+
```

Comment:
```suggestion
​In Front-End development, it's important to consider the best solutions that reduce Memory & CPU consumption in Browser. In fact, a Web page executes only one single thread for all Vue apps, so it's important to avoid adding a lot of stuff to do when mounting a Vue application (https://www.red-gate.com/simple-talk/development/dotnet-development/javascript-single-threaded/[Read more about JS lifecycle]). In fact, when adding heavy computing statements in the page loading phase, it will *pause* mounting all other Vue applications until the computing is finished, because of the single-threaded execution architecture of Web pages. Consequently, having one Vue app that isn't sufficiently performant in a web page, especially for a TopBar application (that is present on all pages), will degrade the rendering performances of the whole page.

```